Kentico/KInspector
# Add support for Kentico 13

### Motivation

This is a large PR (sorry) that adds Kentico 13 support and also includes:

- Actions which modify data (#210)
- Code analysis and formatting for all projects
- On-line marketing macro analysis report (#208)
- Global administrator summary action (#234)
- Sets new repository owner

### Checklist


### How to test

- Clone the repository and follow [these steps](https://github.com/Kentico/KInspector/tree/kentico-13#local-development) to build and run the application locally
- Connect the application to a Kentico 13 instance
- Test each Report and Action to ensure they are functioning
